### ISSUE TITLE
Metrics 3.0/3.1

### ISSUE THREAD

**Issue by lbeuster** [coursera/metrics-datadog#8]
Hi,

are there plans to support metrics 3.0 (com.codahale) or 3.1 (io.dropwizard) with the new package names?

Best regards
Lars

**Comment by torbjornvatn** [comment]
What's the status on this issue?

**Comment by nickdella** [comment]
@torbjornvatn Thanks for the ping. Metrics 3.1.1 was released fairly recently, so I think it's a good time to upgrade. I tested metrics-datadog built with metrics-3.1.1 in my environment and everything seems to work for me.  As for `dropwizard-metrics-datadog`, it compiles, but I'm not able to integration test it myself as I don't use the framework. I'll open a PR and see if @tianx2 has any concerns.
